musicbrainz-dockerfile/Dockerfile

```diff
@@ -28,9 +27,8 @@ RUN curl -sSL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
         python-minimal \
         zlib1g-dev
 
-RUN git clone https://github.com/metabrainz/musicbrainz-server.git musicbrainz-server && \
-    cd musicbrainz-server && \
-    git checkout v-2020-04-13
+ARG MUSICBRAINZ_VERSION=v-2020-04-13
+RUN git clone https://github.com/metabrainz/musicbrainz-server.git -c advice.detachedHead=false -b $MUSICBRAINZ_VERSION musicbrainz-server
```

Comment:
```suggestion
RUN git clone https://github.com/metabrainz/musicbrainz-server.git -c advice.detachedHead=false -b $MUSICBRAINZ_VERSION --depth=1 musicbrainz-server
```

Reply:
Good point. I didn't know this option before. I just added it to all clone commands.